storage/persistent_storage/persistent-storage-vsphere.adoc

```diff
@@ -26,11 +26,11 @@ requested by users.
 
 [IMPORTANT]
 ====
-{product-title} defaults to using an in-tree (non-CSI) plugin to provision vSphere storage.
+{product-title} 4.13 and later provides automatic migration for vSphere in-tree volume plugin to its equivalent CSI driver.
```

Comment:
```suggestion
{product-title} 4.13 and later provides automatic migration for the vSphere in-tree volume plugin to its equivalent CSI driver.
```

Reply:
4.13 is not entirely accurate right? I don't know how to word id - since migration is only enabled for newer clusters in 4.13 and not existing clusters.

Reply:
I've tried to address this in the latest changes. PTAL @gnufied.